build/evergreen_config_generator/__init__.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2018-present MongoDB, Inc.
```

Comment:
Note that I had to add `evergreen_config_generator/*` locally to get the package to work.

Reply:
Sweet, thanks! This will save time for future developers.